README.md

```diff
@@ -112,10 +112,27 @@ Please note that sub documents doesn't have to extend Typegoose. You can still g
 
 `npm install typegoose -S`
 
+## Peer dependencies
+
+In order to make this lib works, you need to install also the peer dependencies:
```

Comment:
should be better: `In order to make this lib work, you also need to install the following peer dependencies:`